repos.yaml

```diff
@@ -77,6 +77,11 @@
   fork: 'mattmoor/eventing-kafka-broker'
   channel: 'eventing-delivery'
   assignees: slinkydeveloper
+- name: 'knative-sandbox/eventing-kafka'
+  meta-organization: 'knative-sandbox'
+  fork: 'mattmoor/eventing-kafka'
```

Comment:
This should be created before merging.

Reply:
it is there: https://github.com/mattmoor/eventing-kafka